### DISCUSSION TITLE
Where do you put the dark mode toggle?

### DISCUSSION THREAD

**Discussion by idc77** [shadcn-ui/ui#5762] [Q&A]
Using the new Sidebar for layout, I wonder where to put the dark/light mode toggle.

Where do you have your dark/light mode button?

![Screenshot from 2024-11-07 16-32-56](https://github.com/user-attachments/assets/8cdd5627-0b2e-4333-970e-40aaa26be5fd)

Looks kind of out of place

**Comment by ao-Barbosa** [comment]
I have the sidebar header as a dropdown where the user has access to some less used links (profile/preferences/some audit things for admin) and quick actions like: Toggle dark/light mode, Pick preferred language, logout, report a bug...
